lib/ocpp/v201/charge_point.cpp

```diff
@@ -2979,6 +3020,30 @@ void ChargePoint::set_connector_operative_status(int32_t evse_id, int32_t connec
     this->evses.at(evse_id)->set_connector_operative_status(connector_id, new_status, persist);
 }
 
+bool ChargePoint::on_try_switch_network_connection_profile(const std::string configuration_slot) {
+    EVLOG_info <<  "=============on_try_switch_network_profile============" << configuration_slot;
+
+    // check if the configuration slot is valid
```

Comment:
you should also call `configure_network_connection_profile_callback` here and follow the 'normal' flow.